examples/model_selection/plot_learning_curve.py

```diff
@@ -77,7 +77,7 @@ def plot_learning_curve(estimator, title, X, y, axes=None, ylim=None, cv=None,
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    train_sizes : array-like of shape (n_ticks,), dtype={int, float}
+    train_sizes : array-like of shape (n_ticks,), dtype=numeric
```

Comment:
As suggested above, not need to put "dtype=numeric" in the oneliner summary of the expected type. The body of the parameter docstring is already explicit enough.

```suggestion
    train_sizes : array-like of shape (n_ticks,)
```

Reply:
I made the change